Frontend/Screens/TaskManagerScreen.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Platform, StyleSheet, Text, View } from 'react-native';
+import TaskManagerComponent from '../Components/TaskManagerComponent';
+
+
+export default class TaskManagerScreen extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <TaskManagerComponent/>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Adding the style to Style.js could be done before merging. Please check the Style.js and add the style over there. Then you can import it.